tests/ui/wrong_self_convention.stderr

```diff
@@ -7,189 +7,189 @@ LL |     fn from_i32(self) {}
    = note: `-D clippy::wrong-self-convention` implied by `-D warnings`
    = help: consider choosing a less ambiguous name
 
-error: methods called `from_*` usually take no self
+error: methods called "from_*" usually take no self
   --> $DIR/wrong_self_convention.rs:24:21
    |
 LL |     pub fn from_i64(self) {}
    |                     ^^^^
    |
    = help: consider choosing a less ambiguous name
 
-error: methods called `as_*` usually take self by reference or self by mutable reference
+error: methods called "as_*" usually take self by reference or self by mutable reference
   --> $DIR/wrong_self_convention.rs:36:15
    |
 LL |     fn as_i32(self) {}
    |               ^^^^
    |
    = help: consider choosing a less ambiguous name
 
-error: methods called `into_*` usually take self by value
+error: methods called "into_*" usually take self by value
   --> $DIR/wrong_self_convention.rs:38:17
    |
 LL |     fn into_i32(&self) {}
    |                 ^^^^^
    |
    = help: consider choosing a less ambiguous name
 
-error: methods called `is_*` usually take self by reference or no self
+error: methods called "is_*" usually take self by reference or no self
   --> $DIR/wrong_self_convention.rs:40:15
    |
 LL |     fn is_i32(self) {}
    |               ^^^^
    |
    = help: consider choosing a less ambiguous name
 
-error: methods called `to_*` usually take self by reference
+error: methods with the following characteristics: ("to_*" and "`self` type is not `Copy`") usually take self by reference
```

Comment:
Oops, I think you copied the `"` signs, which I only used to split the characteristics from the sentence.

Reply:
One thing I'm not sure about now, should to_* have backticks or not?
E.g. this way: 
error: methods with the following characteristics: (to_* and `self` type is not `Copy`) usually take self by reference
Or this: 
error: methods with the following characteristics: (`to_*` and `self` type is not `Copy`) usually take self by reference

Reply:
I think it should be the latter, to drive home that this should match a method name. We could also backtick the `self` in "usually take `self` by reference".